pkg/controllers/status/controller.go

```diff
@@ -554,7 +584,7 @@ func (s *StatusController) latestReplicasetDigests(
 		)
 		if err != nil {
 			keyedLogger.WithValues("cluster-name", clusterName).Error(err, "Failed to get object from cluster")
-			return nil, errors.Wrapf(err, "Failed to get %s %q from cluster %q", targetKind, key, clusterName)
+			continue
```

Comment:
@SOF3 does it make sense to `continue` here, and possibly return an incomplete digest slice? or should we still return an error in this method and have the caller simply skip writing the latestReplicasetDigest annotation when this method errors?

the end goal is to not have an error when getting latestReplicasetDigest prevent status updates.

Reply:
Same as below, just let it error and continue executing upon latestReplicasetDigests error in the caller.

Reply:
ok, this PR has been updated, please take a look.